src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -606,5 +610,11 @@ public void validateOrganisationIsActive(Organisation existingOrganisation) {
         }
     }
 
+    @Override
+    @Transactional
+    public void deleteMultipleAddressOfGivenOrganisation(Set<UUID> idsSet) {
+        contactInformationRepository.deleteByAddressId(idsSet);
```

Comment:
can you confirm if this will also delete dx address if present ?

Reply:
Yes, it will delete dx address as well.

Reply:
ok